### PR TITLE
 Am inceput o calatorie frumoasa ce se va sfarsi cu primul proiect sup…

### PR DESCRIPTION
**Pull request by vasilemaftita**
…er mega ultra interesant

WiP = Work in Progress

### CODE REVIEW

**Review by biggicode — APPROVED** [review]
💯

**Review by Constantin-Matei — APPROVED** [review]
👌

**Review by Adrian01C — APPROVED** [review]

File: web/vasilemaftita/css/css-home/csshome.css [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+body {
+  background-color: lightblue;
+}
+p {
+  font-family: verdana;
+  font-size: 20px;
+  color: brown;
+}
+p {
+  class="p3";
```
Comment (RalucaRbc) [inline web/vasilemaftita/css/css-home/csshome.css]: The way you should be targeting a class from your index.html is using `.` . So delete lines 12-14 and replace them with `.p3 {
  /* style here the class .p3 */
}`

**Review by RalucaRbc — CHANGES_REQUESTED** [review]

**Review by RalucaRbc — APPROVED** [review]

**Comment by vasilemaftita** [comment]
Nice; work

**Review by simonamarcu2 — COMMENTED** [review]
Great 👍

**Pull request merged**